addon/components/file-browser/component.js

```diff
@@ -455,6 +457,7 @@ export default Ember.Component.extend(Analytics, {
             })
             .catch(() => this.get('toast').error(
                 this.get('i18n').t('eosf.components.moveToProject.couldNotMoveFile')
-            ));
+            ))
+            .then(() => this.set('isMoving', false));
```

Comment:
It would be nice to refactor this component to use ember-concurrency tasks instead of chained promises, but that can be another time.